crates/egui_kittest/src/event.rs

```diff
@@ -0,0 +1,179 @@
+use egui::Event::PointerButton;
+use egui::{Event, Modifiers, Pos2};
+use kittest::{ElementState, MouseButton, SimulatedEvent};
+
+#[derive(Default)]
+pub(crate) struct EventState {
+    modifiers: Modifiers,
+    last_mouse_pos: Pos2,
+}
+
+impl EventState {
+    pub fn kittest_event_to_egui(&mut self, event: kittest::Event) -> Option<egui::Event> {
```

Comment:
Does this really need `&mut self`?

Reply:
Yes, because we need to somehow track the modifiers and last mouse position. I aligned the kittest events with winits event types, so the mouse click events don't contain the mouse position.